cilium-cli/connectivity/builder/manifests/client-egress-standalone-dns-proxy.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: cilium.io/v2
+kind: CiliumNetworkPolicy
+metadata:
+  name: client-egress-standalone-dns-proxy
+spec:
+  endpointSelector:
+    matchLabels:
+      kind: client
+  egress:
+  - toPorts:
+    - ports:
+      - port: "53"
+        protocol: UDP
+      - port: "53"
+        protocol: TCP
+      rules:
+        dns:
+        - matchPattern: "one.one.one.one"
```

Comment:
This needs to be parameterized by using `ExternalTarget`. It is not always `one.one.one.one`, even in our workflows.